### PR TITLE
Update Take Action Page With Our Requests and Responding to Pushback

### LINKED ISSUE #221: Integrate Our Requests into Take Action Page
## Description

Integrate Our Requests into Take Action Page right after the letter

image

### LINKED ISSUE #222: Integrate Responding to Pushback into the Take Action Page
## Description

Integrate the responding to pushback cards right before the sign the letter section, and add a conclusion paragraph after this section

image

### PR DESCRIPTION
**Pull request by kennykos**
## Summary

Closes #221, 
no longer closes #222, as issue was depreciated.

Update Take Action Page to have Our Request and Responding to Pushback sections, while making the text on the page more mobile responsive.

## Changes

- Add Our Request
- Add Responding to Pushback
- Add Conclusion Section
- Change padding for text, and make padding responsive to mobile viewing

## Screenshots

image
image
image
image

### CODE REVIEW

File: src/components/accordion/RequestAccordion.tsx [hunk 1]
```diff
@@ -7,15 +7,18 @@ import { NumberCircle, ReadMoreButton } from '../index'
 import type { OurRequestsEntry } from '../../types'
 import { orderedListStyle } from '../../utils/renderer'
 
-const RequestAccordion: FC<{ content: OurRequestsEntry }> = ({ content }) => {
+const RequestAccordion: FC<{
+  content: OurRequestsEntry
+  color?: 'lightBlue'
```
Comment (zorazrr) [inline src/components/accordion/RequestAccordion.tsx:12]:
```suggestion
  color?: string
```

File: src/pages/take-action/index.tsx [hunk 2]
```diff
@@ -4,28 +4,59 @@ import { scroller } from 'react-scroll'
...
+  ourRequestsEntries,
+  respondingToPushbackEntries,
+}) => {
   const navItems = [
     { path: 'openLetter', text: 'Open Letter' },
+    { path: 'ourRequests', text: 'Our Requests' },
+    { path: 'respondingToPushback', text: 'Responding to Pushback' },
     { path: 'signLetter', text: 'Sign the Letter' },
     // { path: 'signatories', text: 'Signatories' },
     // { path: 'momentum', text: 'Continue the Momentum' },
```
Comment (zorazrr) [inline src/pages/take-action/index.tsx:62]: Maybe remove if unused, otherwise add a `// TODO:`

File: src/sections/index.tsx [hunk 3]
```diff
@@ -41,6 +44,9 @@ export {
   UniversityInvestments,
   WhatWarmingMeans,
   OpenLetter as Letter,
+  LetterOurRequest as LetterRequests,
+  RespondingToPushback,
+  OpenLetterConclusion as Conclusion,
```
Comment (zorazrr) [inline src/sections/index.tsx:49]: Not sure why the `as` here are necessary?

File: src/sections/take-action/conclusion.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+const OpenLetterConclusion: FC<LetterProps> = ({ openLetter }) => {
+  const { isLoading } = api.signatory.getSignatoriesCount.useQuery()
+  if (isLoading) {
+    return (
+      <div className="flex flex-col items-center px-12">
+        <Spinner color="info" />
+      </div>
+    )
+  }
```
Comment (zorazrr) [inline src/sections/take-action/conclusion.tsx:21]: Is this query necessary here? Seems like no data is used in this page.

File: src/sections/take-action/responding-to-pushback.tsx
Comment (zorazrr) [inline src/sections/take-action/responding-to-pushback.tsx:1]: This is unrelated to the code but is it necessary to have two duplicate sections? Definitely not the expert here so just wondering bc of the amount of duplicate code.

**Review by zorazrr — CHANGES_REQUESTED** [review]
Some redundant elements

File: src/sections/take-action/responding-to-pushback.tsx
Comment (kennykos) [reply src/sections/take-action/responding-to-pushback.tsx:1]: I added conclusion to the letter class, let me know if you like that better.

Comment (kennykos) [reply src/sections/take-action/responding-to-pushback.tsx:1]: Ok we settled on just having the conclusion before the requests so everything is back to normal

Comment (zorazrr) [inline src/sections/take-action/responding-to-pushback.tsx:1]: If we are not using this should we delete this component?

Comment (kennykos) [reply src/sections/take-action/responding-to-pushback.tsx:1]: agreed, fixed it!

Comment (kennykos) [reply src/sections/take-action/responding-to-pushback.tsx:1]: done

File: src/pages/take-action/index.tsx [hunk 5]
```diff
@@ -14,27 +25,40 @@ export const getServerSideProps = async () => {
   const takeActionPageEntries = takeActionEntries['takeActionPage']
   const momentumEntries = takeActionEntries['continueTheMomentumEntries']
 
+  const homeEntries = await contentClient.getAllHomePageEntries()
+  const info = homeEntries['info']
+  const ourRequestsEntries = homeEntries['request']
+  const respondingToPushbackEntries = homeEntries['response']
```
Comment (zorazrr) [inline src/pages/take-action/index.tsx:31]: Laaaaast thing maybe delete the entries here as well

Comment (kennykos) [reply src/pages/take-action/index.tsx]: ok, got the last one, hopefully that's all :pray:

File: src/pages/take-action/index.tsx [hunk 6]
```diff
@@ -14,27 +25,38 @@ export const getServerSideProps = async () => {
...
       momentumEntries,
     },
   }
 }
 
 interface TakeActionPageProps {
+  info: HomePageInfo
   takeActionPageEntries: TakeActionPage
+  ourRequestsEntries: OurRequestsEntry[]
+  respondingToPushbackEntries: RefuteResponseEntry[]
```
Comment (zorazrr) [inline src/pages/take-action/index.tsx]:
```suggestion
```

File: src/pages/take-action/index.tsx [hunk 7]
```diff
@@ -4,8 +4,19 @@ import { scroller } from 'react-scroll'
...
+  Letter,
+  LetterOurRequest,
+  SignLetter,
+  Signatories,
+  Momentum,
+} from '../../sections'
+import type {
+  OurRequestsEntry,
+  HomePageInfo,
+  RefuteResponseEntry,
```
Comment (zorazrr) [inline src/pages/take-action/index.tsx]:
```suggestion
```

**Review by zorazrr — APPROVED** [review]

**Pull request merged**